ctapipe/coordinates/transforms.py

```diff
@@ -0,0 +1,36 @@
+from astropy.coordinates import SkyCoord, AltAz
+from ..coordinates import CameraFrame
+import astropy.units as u
+
+
+
+def event_pos_in_camera(event, tel_id, horizon_frame=AltAz()):
+    """
```

Comment:
this function name makes it seem like it's the position of an event in the camera, not the source position.  Is that really what you want ?  Also this will be *very* slow if you plot many events, since it constructs the camera frame each time. 

I think it may be better to store a constructed frame in either the CameraDisplay or the CameraGeometry so that it can be re-used.  Something like `display.frame`, then you can just update its' parameters as needed (if the pointing changes).    This also reminds me to clean up the place where pointings are stored, and to use SkyCoords for positions.

Reply:
>  Also this will be very slow if you plot many events, since it constructs the camera frame each time.

Instantiating an `AltAz` frame takes < 1ms on my machine. Why do you think, this will slow down plotting?

Reply:
CameraFrame instantiation takes 200 µs